Remotion/ObjectBinding/Validation/BusinessObjectValidationResult.cs

```diff
@@ -52,26 +52,15 @@ private static IEnumerable<BusinessObjectValidationFailure> CreateBusinessObject
 
       switch (validationFailure)
       {
-        case PropertyValidationFailure propertyValidationFailure
-            when validationFailure.ValidatedObject is IBusinessObject validatedBusinessObject
-                 && GetBusinessObjectProperty(validatedBusinessObject, propertyValidationFailure.ValidatedProperty) is { } validatedBusinessObjectProperty:
-        {
-          return EnumerableUtility.Singleton(
-              BusinessObjectValidationFailure.CreateForBusinessObjectProperty(
-                  propertyValidationFailure.LocalizedValidationMessage,
-                  validatedBusinessObject,
-                  validatedBusinessObjectProperty));
-        }
-
-        case ObjectValidationFailure objectValidationFailure
+        case ValidationFailure objectValidationFailure
```

Comment:
The current updated solutions looks okay. I want to try more simplification and see if that can make things better. we should do this together.